src/cmd/builtin/uptime.c

```diff
@@ -219,8 +219,7 @@ static void append_count_if (char *buf,
         int n = strlen (buf);
         snprintf (buf + n,
```

Comment:
commit message, minor english, "a comma and space", not plural spaces

Reply:
It's actually one comma and two spaces.